packages/domain-picker/src/components/domain-suggestion-item/suggestion-item-wrapper.tsx

```diff
@@ -6,7 +6,7 @@ import * as React from 'react';
 /**
  * Internal dependencies
  */
-import type { SUGGESTION_ITEM_TYPE } from 'src';
+import { SUGGESTION_ITEM_TYPE, SUGGESTION_ITEM_TYPE_BUTTON } from './suggestion-item';
```

Comment:
What do you think about this, more explicit, though verbose, approach?

```suggestion
import type { SUGGESTION_ITEM_TYPE } from './suggestion-item';
import { SUGGESTION_ITEM_TYPE_BUTTON } from './suggestion-item';
```

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/51235/commits/693ce38a124aa02293aeb15367692937c823995f